src/dlgConnectionProfiles.cpp

```diff
@@ -42,21 +42,7 @@
 using namespace std::chrono_literals;
 
 dlgConnectionProfiles::dlgConnectionProfiles(QWidget* parent)
-: QDialog(parent)
-, validName()
-, validUrl()
-, validPort()
-, mProfileList(QStringList())
-, offline_button(nullptr)
-, connect_button(nullptr)
-, delete_profile_lineedit(nullptr)
-, delete_button(nullptr)
-, mDefaultGames({"3Kingdoms", "3Scapes", "Aardwolf", "Achaea", "Aetolia",
-                 "Avalon.de", "BatMUD", "Clessidra", "Fierymud", "Imperian", "Luminari",
-                 "Lusternia", "Materia Magica", "Midnight Sun 2", "Realms of Despair",
-                 "Reinos de Leyenda", "StickMUD", "WoTMUD", "ZombieMUD", "Carrion Fields",
-                 "Cleft of Dimensions", "CoreMUD", "God Wars II", "Slothmud",
-                 "Legends of the Jedi"})
+: QDialog(parent), validName(), validUrl(), validPort(), offline_button(nullptr), connect_button(nullptr), delete_profile_lineedit(nullptr), delete_button(nullptr)
```

Comment:
:question: 

Is there a way to prevent clang-tidy from tidying this up so much - to my mind it is actually more convenient to have each member being initialized on a separate line because any changes are clearer in git and one doesn't have to scroll along long lines to read it all...

Of course all but `QDialog` and `mDefaultGames` can be better done in the header file nowadays (and maybe even `mDefaultGames` might be doable there - or, well in the `mudlet.h` header I guess now, as well...!)

Reply:
Ah remember we're slowly moving away from initializing like this anyway...

but yeah, there is typically an option in the to only format what you changed, not the whole file.